src/main/java/tools/jackson/databind/cfg/EnumFeature.java

```diff
@@ -34,7 +34,54 @@ public enum EnumFeature implements DatatypeFeature
      *
      * @since 2.15
      */
-    WRITE_ENUMS_TO_LOWERCASE(false);
+    WRITE_ENUMS_TO_LOWERCASE(false),
+
+
+    /**
+     * Feature that determines standard serialization mechanism used for
+     * Enum values: if enabled, return value of <code>Enum.toString()</code>
+     * is used; if disabled, return value of <code>Enum.name()</code> is used.
+     *<p>
+     * Note: this feature should usually have same value
+     * as {@link DeserializationFeature#READ_ENUMS_USING_TO_STRING}.
+     *<p>
```

Comment:
Could you also add a short paragraph on move from `SerializationFeature`, similar to how `DateTimeFeature` entries have it. Something like:

```
     *<p>
     * Feature used to be one of {@link tools.jackson.databind.SerializationFeature}s
     * in Jackson 2.x but was moved here in 3.0.
```